calour/tests/test_experiment.py

```diff
@@ -102,6 +104,59 @@ def test_reorder_round_trip(self):
 
         assert_experiment_equal(new, exp)
 
+    def test_copy_experiment(self):
+        exp = copy(self.test1)
+        assert_experiment_equal(exp, self.test1)
+        exp = deepcopy(self.test1)
+        assert_experiment_equal(exp, self.test1)
+
+    def test_get_data(self):
+        # default - do not modify the data
+        exp = deepcopy(self.test1)
+        data = exp.get_data()
+        self.assertTrue(sparse.issparse(data))
+        self.assertEqual(data.sum(), exp.data.sum())
+        # test it's not a copy but inplace
+        data[0, 0] = -1000
+        self.assertEqual(data.sum(), exp.data.sum())
+
+        # lets force it to copy
+        exp = deepcopy(self.test1)
+        data = exp.get_data(copy=True)
+        self.assertTrue(sparse.issparse(data))
+        self.assertEqual(data.sum(), exp.data.sum())
+        # test it's not a copy but inplace
+        data[0, 0] = -1000
+        self.assertNotEqual(data.sum(), exp.data.sum())
```

Comment:
again for all the followings: 

use self.assertIs and self.assertIsNot to check if they are the same object or not